Mathlib/Topology/Algebra/Group/Basic.lean

```diff
@@ -178,6 +178,8 @@ over `M`, for example, is obtained by requiring the instances `AddGroup M` and
 class ContinuousNeg (G : Type u) [TopologicalSpace G] [Neg G] : Prop where
   continuous_neg : Continuous fun a : G => -a
 #align has_continuous_neg ContinuousNeg
+-- Porting note: added
+attribute [continuity] ContinuousNeg.continuous_neg
```

Comment:
Do you understand why `continuity` on `Continuous.neg` is not enough?

Reply:
I think it is because `aesop` ends up going using this instead of `Continuous.neg continuous_id`. In particular, the inner term in `exp (-x)` looks like just a `Neg.neg` application as opposed to a `Neg.neg \comp f`.